scene/gui/text_edit.h

```diff
@@ -632,6 +633,10 @@ class TextEdit : public Control {
 	int get_indent_level(int p_line) const;
 	bool is_line_comment(int p_line) const;
 
+	bool is_line_empty_or_comment(int p_line) const;
```

Comment:
I'm pretty sure `TextEdit`, a simple textarea control, should not care, much less know, what is a comment. Especially since different programming languages can have different formats for comments.